lib/inspec/profile.rb

```diff
@@ -124,6 +126,10 @@ def version
       metadata.params[:version]
     end
 
+    def cache_resources?
+      metadata.params.fetch(:cache_resources, false)
```

Comment:
I think we should not make the caching part of the inspec.yml since this is static content. From my perspective, this  needs to be a flag for inspec cli where by default caching is turned off